<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font: 14pt &quot;Terminal&quot;;</string>
    </property>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>731</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color:rgb(216, 249, 255)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_titulo">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>491</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Impact</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    border: 4px solid #5DADE2;
    background-color: #D8F9FF;  /* Azul claro del fondo de la imagen */
    color: black;               /* Texto en negro */
    padding: 10px;              /* Espaciado interno */
    border-radius: 50px;        /* Bordes redondeados */
}

QLabel {
    font-family: &quot;Impact&quot;;
    font-size: 20pt;
    qproperty-alignment: 'AlignCenter';
}

</string>
     </property>
     <property name="text">
      <string>Traduccion a Ruby</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>711</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit {
    border: 4px outset #5DADE2;  /* Borde con efecto elevado */
    background-color: white;  /* Fondo blanco */
    padding: 5px;  /* Espaciado interno */
    font-size: 14pt; 
}

</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>imagenes/ruby.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
